### DISCUSSION TITLE
Stucked at  "npx fvst infra init"

### DISCUSSION THREAD

**Discussion by zeedevio** [fvst-dev/fvst-monorepo#95] [Q&A]
When I run the command, it stucks at Checking dependencies... and never completes is there a set of manual steps I can complete ?

**Comment by darynmitchell** [comment]
You can throw some `console.log()`s into _init.ts_ to see how far it gets. If you identify the command which is causing it, then try running the command manually. If might interactively show you what need to be done to resolve.

For instance, if the place where it’s getting stuck is in `getBillingAccounts()` at
```js
    const [stdout] = safeExec('gcloud beta billing accounts list --format json');
```
then try running that via the command line, 
```
$ gcloud beta billing accounts list --format json

You do not currently have this command group installed.  Using it 
requires the installation of components: ...

Do you want to install? y/n
```

**Reply by zeedevio** [reply]
The CLI Commands were outdated or at least it did not work with what I wanted. What I did was build my version of CLI and today use it to setup nest and next project's on GCP. Thanks for reaching out though.
